Blogplace.Tests.Integration/Tests/UsersTests.cs

```diff
@@ -0,0 +1,67 @@
+﻿using Blogplace.Web.Domain;
+using FluentAssertions;
+using System.Net.Http.Json;
+using System.Text.RegularExpressions;
+
+namespace Blogplace.Tests.Integration.Tests;
+public class UsersTests : TestBase
+{
+    [Test]
+    public async Task GetById_ShouldBePublic()
+    {
+        //Arrange
+        var client = this.CreateClient(withSession: false);
+        var userClient = await this.RegisterNewUser(client);
+        var user = await this.GetMe(userClient);
+        var anonymousClient = client.WithoutToken();
+        var request = new GetUserByIdRequest(user.Id);
+
+        //Act
+        var response = await anonymousClient.PostAsync($"{this.urlBaseV1}/Users/GetById", request);
+
+        //Assert
+        response.StatusCode.Should().Be(System.Net.HttpStatusCode.OK);
+        var result = await response.Content.ReadFromJsonAsync<GetUserByIdResponse>();
+        result!.User.Should().BeEquivalentTo(user);
+    }
+
+    [Test]
+    public async Task Update_ShouldUpdateOwnUsername()
+    {
+        //Arrange
+        var newUsername = Guid.NewGuid().ToString();
+        var client = this.CreateClient(withSession: false);
+        var userClient = await this.RegisterNewUser(client);
+        var oldUser = await this.GetMe(userClient);
+
+        var request = new UpdateUserRequest(newUsername);
+
+        //Act
+        await userClient.PostAsync($"{this.urlBaseV1}/Users/Update", request);
+        var updatedUser = await this.GetMe(userClient);
+
+        //Assert
+        oldUser.Username.Should().NotBe(newUsername);
+        updatedUser.Username.Should().Be(newUsername);
+    }
+
+    private async Task<ApiClient> RegisterNewUser(ApiClient baseClient)
```

Comment:
to później można wynieść do `TestBase` bo pewnie będzie często używane w innych testach